### PR TITLE
Type Definition

### LINKED ISSUE #67: types not found for mobx-devtools-mst
I'm attempting to use `mobx-devtools-mst` in typescript and I'm not seeing types. I tried to install `@types/mobx-devtools-mst` but I see types have been removed from that as it says that types are provided. I've got `0.9.21` installed and don't see any types however.

### PR DESCRIPTION
**Pull request by kuasha420**
Fixes #67

### CODE REVIEW

File: packages/mobx-devtools-mst/types/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+declare module 'mobx-devtools-mst' {
+  export default function makeInspectable(state: object): void;
```
Comment (epodgaetskiy) [inline packages/mobx-devtools-mst/types/index.d.ts]:
```suggestion
  type Root = object;
  export default function makeInspectable(root: Root): Root;
```

**Review by epodgaetskiy — CHANGES_REQUESTED** [review]
Please merge master, and remove everything with dependencies.

**Comment by kuasha420** [comment]
@epodgaetskiy Everything ok? Monorepo always trips me off..

BTW, eslint is failing on precommit hook, had no --no-verify it. The error is about unresolved electron import on `mobx-devtools/packages/playground/electron/background.js`

**Comment by epodgaetskiy** [comment]
> @epodgaetskiy Everything ok? Monorepo always trips me off..
> 
> BTW, eslint is failing on precommit hook, had no --no-verify it. The error is about unresolved electron import on `mobx-devtools/packages/playground/electron/background.js`

Thank you. I fixed it. Please merge master.

**Comment by kuasha420** [comment]
@epodgaetskiy Done.

**Pull request merged**